invenio_opendata/testsuite/data/cms/cms-tools-dimuon-spectrum-2010.xml

```diff
@@ -83,16 +83,16 @@
       <subfield code="b">Research</subfield>
     </datafield>
     <datafield tag="FFT" ind1=" " ind2=" ">
-      <subfield code="a">/tmp/opendata.cern.ch-fft-file-cache/cms-tools-dimuon-spectrum-2010/BuildFile.xml</subfield>
+      <subfield code="a">/tmp/opendata.cern.ch-fft-file-cache/cms-tools-di-muon-spectrum-2010/BuildFile.xml</subfield>
     </datafield>
     <datafield tag="FFT" ind1=" " ind2=" ">
-      <subfield code="a">/tmp/opendata.cern.ch-fft-file-cache/cms-tools-dimuon-spectrum-2010/DemoAnalyzer.cc</subfield>
+      <subfield code="a">/tmp/opendata.cern.ch-fft-file-cache/cms-tools-di-muon-spectrum-2010/DemoAnalyzer.cc</subfield>
```

Comment:
Note that we cannot so easily change `di-muon` in the file names, we'd have to rename the files and adapt populating scripts... For now I'll simply cherry-pick your PR to keep only textual changes.